draft-detecting-unwanted-location-trackers.md

```diff
@@ -255,31 +255,32 @@ Data fragmentation and reassembly is not defined in this document; therefore, th
 In other words, all opcode response data must fit within a single write operation.
 
 ## Accessory Information
-The following accessory information MUST be persistent through the lifetime of the accessory: [Product data](#product-data), [Manufacturer name](#manufacturer-name), [Model name](#model-name), [Accessory category](#accessory-category), [Accessory capabilities](#accessory-capabilities), [Firmware version](#firmware-version), and [Network ID](#network-id).
-
-
+The following accessory information MUST be persistent through the lifetime of the accessory: [Product data](#product-data), [Manufacturer name](#manufacturer-name), [Model name](#model-name), [Accessory category](#accessory-category), [Accessory capabilities](#accessory-capabilities), [Network ID](#network-id), [Battery Type](#battery-type), and [Battery State](#battery-state).
```

Comment:
Battery state must be removed from here as it cannot be persistent throughout the accessory lifetime.